crates/proto/src/udp/udp_stream.rs

```diff
@@ -67,6 +67,8 @@ where
     async fn send_to(&self, buf: &[u8], target: SocketAddr) -> io::Result<usize> {
         futures_util::future::poll_fn(|cx| self.poll_send_to(cx, buf, target)).await
     }
+
+    async fn writable(&self) -> io::Result<()>;
```

Comment:
This guy needs a rustdoc comment to fix a docs build err:

```suggestion
    /// Check whether the stream is currently writable.
    async fn writable(&self) -> io::Result<()>;
```